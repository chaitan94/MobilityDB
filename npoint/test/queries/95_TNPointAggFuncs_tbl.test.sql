-------------------------------------------------------------------------------

SELECT numInstants(tcount(inst)) FROM tbl_tnpointinst;
SELECT numInstants(wcount(inst, '1 hour')) FROM tbl_tnpointinst;
SELECT numInstants(tcentroid(inst)) FROM tbl_tnpointinst;
SELECT k%10, numInstants(tcount(inst)) FROM tbl_tnpointinst GROUP BY k%10 ORDER BY k%10;
SELECT k%10, numInstants(wcount(inst, '1 hour')) FROM tbl_tnpointinst GROUP BY k%10 ORDER BY k%10;
SELECT k%10, numInstants(tcentroid(inst)) FROM tbl_tnpointinst GROUP BY k%10 ORDER BY k%10;

SELECT numInstants(tcount(ti)) FROM tbl_tnpointi;
SELECT numInstants(wcount(ti, '1 hour')) FROM tbl_tnpointi;
SELECT numInstants(tcentroid(ti)) FROM tbl_tnpointi;
SELECT k%10, numInstants(tcount(ti)) FROM tbl_tnpointi GROUP BY k%10 ORDER BY k%10;
SELECT k%10, numInstants(tcount(ti)) FROM tbl_tnpointi GROUP BY k%10 ORDER BY k%10;
SELECT k%10, numInstants(tcentroid(ti)) FROM tbl_tnpointi GROUP BY k%10 ORDER BY k%10;

SELECT numSequences(tcount(seq)) FROM tbl_tnpointseq;
SELECT numSequences(wcount(seq, '1 hour')) FROM tbl_tnpointseq;
SELECT numSequences(tcentroid(seq)) FROM tbl_tnpointseq;
SELECT k%10, numSequences(tcount(seq)) FROM tbl_tnpointseq GROUP BY k%10 ORDER BY k%10;
SELECT k%10, numSequences(wcount(seq, '1 hour')) FROM tbl_tnpointseq GROUP BY k%10 ORDER BY k%10;
SELECT k%10, numSequences(tcentroid(seq)) FROM tbl_tnpointseq GROUP BY k%10 ORDER BY k%10;

SELECT numSequences(tcount(ts)) FROM tbl_tnpoints;
SELECT numSequences(wcount(ts, '1 hour')) FROM tbl_tnpoints;
SELECT numSequences(tcentroid(ts)) FROM tbl_tnpoints;
SELECT k%10, numSequences(tcount(ts)) FROM tbl_tnpoints GROUP BY k%10 ORDER BY k%10;
SELECT k%10, numSequences(wcount(ts, '1 hour'))) FROM tbl_tnpoints GROUP BY k%10 ORDER BY k%10;
SELECT k%10, numSequences(tcentroid(ts)) FROM tbl_tnpoints GROUP BY k%10 ORDER BY k%10;

-------------------------------------------------------------------------------
