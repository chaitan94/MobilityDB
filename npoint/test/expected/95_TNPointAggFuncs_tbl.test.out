SELECT numInstants(tcount(inst)) FROM tbl_tnpointinst;
 numinstants 
-------------
         100
(1 row)

SELECT numInstants(wcount(inst, '1 hour')) FROM tbl_tnpointinst;
 numinstants 
-------------
         201
(1 row)

SELECT numInstants(tcentroid(inst)) FROM tbl_tnpointinst;
 numinstants 
-------------
         100
(1 row)

SELECT k%10, numInstants(tcount(inst)) FROM tbl_tnpointinst GROUP BY k%10 ORDER BY k%10;
 ?column? | numinstants 
----------+-------------
        0 |          10
        1 |          10
        2 |          10
        3 |          10
        4 |          10
        5 |          10
        6 |          10
        7 |          10
        8 |          10
        9 |          10
(10 rows)

SELECT k%10, numInstants(wcount(inst, '1 hour')) FROM tbl_tnpointinst GROUP BY k%10 ORDER BY k%10;
 ?column? | numinstants 
----------+-------------
        0 |          20
        1 |          20
        2 |          20
        3 |          20
        4 |          20
        5 |          20
        6 |          20
        7 |          20
        8 |          20
        9 |          20
(10 rows)

SELECT k%10, numInstants(tcentroid(inst)) FROM tbl_tnpointinst GROUP BY k%10 ORDER BY k%10;
 ?column? | numinstants 
----------+-------------
        0 |          10
        1 |          10
        2 |          10
        3 |          10
        4 |          10
        5 |          10
        6 |          10
        7 |          10
        8 |          10
        9 |          10
(10 rows)

SELECT numInstants(tcount(ti)) FROM tbl_tnpointi;
 numinstants 
-------------
         476
(1 row)

SELECT numInstants(wcount(ti, '1 hour')) FROM tbl_tnpointi;
 numinstants 
-------------
        1328
(1 row)

SELECT numInstants(tcentroid(ti)) FROM tbl_tnpointi;
 numinstants 
-------------
         476
(1 row)

SELECT k%10, numInstants(tcount(ti)) FROM tbl_tnpointi GROUP BY k%10 ORDER BY k%10;
 ?column? | numinstants 
----------+-------------
        0 |          34
        1 |          64
        2 |          51
        3 |          36
        4 |          44
        5 |          44
        6 |          45
        7 |          68
        8 |          48
        9 |          42
(10 rows)

SELECT k%10, numInstants(tcount(ti)) FROM tbl_tnpointi GROUP BY k%10 ORDER BY k%10;
 ?column? | numinstants 
----------+-------------
        0 |          34
        1 |          64
        2 |          51
        3 |          36
        4 |          44
        5 |          44
        6 |          45
        7 |          68
        8 |          48
        9 |          42
(10 rows)

SELECT k%10, numInstants(tcentroid(ti)) FROM tbl_tnpointi GROUP BY k%10 ORDER BY k%10;
 ?column? | numinstants 
----------+-------------
        0 |          34
        1 |          64
        2 |          51
        3 |          36
        4 |          44
        5 |          44
        6 |          45
        7 |          68
        8 |          48
        9 |          42
(10 rows)

SELECT numSequences(tcount(seq)) FROM tbl_tnpointseq;
 numsequences 
--------------
          100
(1 row)

SELECT numSequences(wcount(seq, '1 hour')) FROM tbl_tnpointseq;
 numsequences 
--------------
          100
(1 row)

SELECT numSequences(tcentroid(seq)) FROM tbl_tnpointseq;
 numsequences 
--------------
          100
(1 row)

SELECT k%10, numSequences(tcount(seq)) FROM tbl_tnpointseq GROUP BY k%10 ORDER BY k%10;
 ?column? | numsequences 
----------+--------------
        0 |           10
        1 |           10
        2 |           10
        3 |           10
        4 |           10
        5 |           10
        6 |           10
        7 |           10
        8 |           10
        9 |           10
(10 rows)

SELECT k%10, numSequences(wcount(seq, '1 hour')) FROM tbl_tnpointseq GROUP BY k%10 ORDER BY k%10;
 ?column? | numsequences 
----------+--------------
        0 |           10
        1 |           10
        2 |           10
        3 |           10
        4 |           10
        5 |           10
        6 |           10
        7 |           10
        8 |           10
        9 |           10
(10 rows)

SELECT k%10, numSequences(tcentroid(seq)) FROM tbl_tnpointseq GROUP BY k%10 ORDER BY k%10;
 ?column? | numsequences 
----------+--------------
        0 |           10
        1 |           10
        2 |           10
        3 |           10
        4 |           10
        5 |           10
        6 |           10
        7 |           10
        8 |           10
        9 |           10
(10 rows)

SELECT numSequences(tcount(ts)) FROM tbl_tnpoints;
 numsequences 
--------------
          483
(1 row)

SELECT numSequences(wcount(ts, '1 hour')) FROM tbl_tnpoints;
 numsequences 
--------------
          502
(1 row)

SELECT numSequences(tcentroid(ts)) FROM tbl_tnpoints;
 numsequences 
--------------
          490
(1 row)

SELECT k%10, numSequences(tcount(ts)) FROM tbl_tnpoints GROUP BY k%10 ORDER BY k%10;
 ?column? | numsequences 
----------+--------------
        0 |           65
        1 |           37
        2 |           51
        3 |           43
        4 |           51
        5 |           34
        6 |           48
        7 |           58
        8 |           57
        9 |           41
(10 rows)

SELECT k%10, numSequences(wcount(ts, '1 hour'))) FROM tbl_tnpoints GROUP BY k%10 ORDER BY k%10;
ERROR:  syntax error at or near ")"
LINE 1: SELECT k%10, numSequences(wcount(ts, '1 hour'))) FROM tbl_tn...
                                                       ^
SELECT k%10, numSequences(tcentroid(ts)) FROM tbl_tnpoints GROUP BY k%10 ORDER BY k%10;
 ?column? | numsequences 
----------+--------------
        0 |           65
        1 |           37
        2 |           51
        3 |           43
        4 |           51
        5 |           34
        6 |           48
        7 |           58
        8 |           57
        9 |           41
(10 rows)

